src/test/java/software/amazon/event/kafkaconnector/DetailTypeMapperTest.java

```diff
@@ -4,13 +4,16 @@
  */
 package software.amazon.event.kafkaconnector;
 
+import static org.apache.kafka.connect.data.Schema.STRING_SCHEMA;
 import static org.hamcrest.CoreMatchers.is;
 import static org.hamcrest.MatcherAssert.assertThat;
 
 import java.util.Map;
+import org.apache.kafka.connect.sink.SinkRecord;
 import org.junit.jupiter.api.Test;
+import software.amazon.event.kafkaconnector.mapping.DefaultDetailTypeMapper;
 
-public class EventBridgeSinkConfigTest {
+public class DetailTypeMapperTest {
```

Comment:
Do you envision this to be purely for `DefaultDetailTypeMapper` or to add additional tests in future as well?

Reply:
If we provide additional mapper out of the box then I would add a separate new test class.